domain-models-runtime/src/test/java/org/folio/rest/persist/ddlgen/SchemaMakerTest.java

```diff
@@ -56,11 +56,27 @@ private SchemaMaker schemaMaker(String tenant, String module, TenantOperation mo
     return schemaMaker;
   }
 
+  @Test
+  public void testCreate() throws IOException, TemplateException {
+    SchemaMaker schemaMaker = schemaMaker("harvard", "circ", TenantOperation.CREATE,
+        "mod-foo-18.2.3", null, "templates/db_scripts/schemaWithAudit.json");
+    String result = schemaMaker.generateCreate();
+    assertThat(result, containsString("CREATE SCHEMA harvard_circ"));
+  }
+
+  @Test
+  public void testCreateIndexesOnly() throws IOException, TemplateException {
+    SchemaMaker schemaMaker = schemaMaker("harvard", "circ", TenantOperation.CREATE,
+        "mod-foo-18.2.3", null, "templates/db_scripts/schemaWithAudit.json");
+    String result = schemaMaker.generateIndexesOnly();
+    assertThat(result, containsString("CREATE INDEX IF NOT EXISTS audit_"));
```

Comment:
Can you assert that it does not contain "CREATE SCHEMA"?

Reply:
yes